common/views/components/PopupDialog/PopupDialog.js

```diff
@@ -254,6 +264,7 @@ const PopupDialog = ({ children, openButtonText, cta }: Props) => {
         </PopupDialogOpen>
         <PopupDialogWindow ref={dialogWindowRef} isActive={isActive}>
           <PopupDialogClose
+            isKeyboard={isKeyboard}
```

Comment:
could this not be in the actual dialogue itself?

Reply:
I don't think I understand the question. We get `isKeyboard` from the `AppContext` and use it here to conditionally show/hide the close button focus styles.

Reply:
I was wondering if the `PopupDialogClose` should either have the `AppContext` itself, or this component should handle the show/hiding?

Reply:
`PopupDialogClose` is a styled-component rather than a fully-fledged component (so doesn't have the capacity to handle that stuff itself). Does that answer the question?